wg-ecosystem/README.md

```diff
@@ -63,22 +63,38 @@ These repos are sorted alphabetically, their order does not reflect that any of
 
 ### Electron Organization
 
+* [`electron/algolia-indices`](https://github.com/electron/algolia-indices)
 * [`electron/apps`](https://github.com/electron/apps)
+* [`electron/asar`](https://github.com/electron/asar)
+* [`electron/docs-parser`](https://github.com/electron/docs-parser)
+* [`electron/download-stats`](https://github.com/electron/download-stats)
 * [`electron/electron-api-demos`](https://github.com/electron/electron-api-demos)
+* [`electron/electron-api-historian`](https://github.com/electron/electron-api-historian)
 * [`electron/electron-docs-linter`](https://github.com/electron/electron-docs-linter)
+* [`electron/electron-docs`](https://github.com/electron/electron-docs)
+* [`electron/electron-issues`](https://github.com/electron/electron-issues)
 * [`electron/electron-notarize`](https://github.com/electron/electron-notarize)
 * [`electron/electron-osx-sign`](https://github.com/electron/electron-osx-sign)
 * [`electron/electron-packager`](https://github.com/electron/electron-packager)
+* [`electron/electron-quick-start-typescript`](https://github.com/electron/electron-quick-start-typescript)
 * [`electron/electron-quick-start`](https://github.com/electron/electron-quick-start)
 * [`electron/electron-rebuild`](https://github.com/electron/electron-rebuild)
-* [`electron/electron-typescript-definitions`](https://github.com/electron/electron-typescript-definitions)
 * [`electron/electronjs.org`](https://github.com/electron/electronjs.org)
 * [`electron/fiddle`](https://github.com/electron/fiddle)
 * [`electron/get`](https://github.com/electron/get)
+* [`electron/hubdown`](https://github.com/electron/hubdown)
 * [`electron/i18n`](https://github.com/electron/i18n)
+* [`electron/node-minidump`](https://github.com/electron/node-minidump)
 * [`electron/node-rcedit`](https://github.com/electron/node-rcedit)
 * [`electron/rcedit`](https://github.com/electron/rcedit)
 * [`electron/releases`](https://github.com/electron/releases)
+* [`electron/remote`](https://github.com/electron/remote)
+* [`electron/semantic-release-npm-cfa`](https://github.com/electron/semantic-release-npm-cfa)
+* [`electron/simple-samples`](https://github.com/electron/simple-samples)
```

Comment:
Should we hide this because the repo has been archived?

Reply:
That repo still have an entry in `.permission` about that `wg-ecosystem` have to write access, I can add logic for skip archive repos if we prefer to not show them, or just remove `ecosystem-wg` access from `.permissions`  what's seems to prefer for me

Reply:
Sounds good to me! In any case, write access to an archived repo means nothing without admin permissions. 🤷